tests/global-teardown.ts

```diff
@@ -1,11 +1,17 @@
-import Strapi from "@strapi/strapi";
 import { cleanUpDatabase } from "../init/common";
 
 async function globalTeardown() {
-    if (process.env.TEST_CONTAINER !== 'true') {
-        await Strapi();
-        console.log('Deleting test database...');
-        await cleanUpDatabase(strapi);
+    if (process.env.TEST_CONTAINER === 'true') {
+        const container = global.postgresContainer;
```

Comment:
Using `global` variables allows to share started Strapi and Container between setup and teardown routine.